package.json

```diff
@@ -14,6 +14,9 @@
     "@graphql-codegen/visitor-plugin-common": "^1.17.0",
     "auto-bind": "~4.0.0"
   },
+  "peerDependencies": {
+    "graphql": "^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0"
```

Comment:
Not sure which versions you want here so I took the ones from https://github.com/dotansimha/graphql-code-generator/blob/ef0e7c411dff26fde60a60b5d21421f0baa92ff2/packages/graphql-codegen-cli/package.json#L108

Reply:
The peer dependency from gql codegen is perfect, thank you very much for the contribution